civic.json

```diff
@@ -0,0 +1,8 @@
+{
+    "status": "Beta", 
+    "tags": [
+        "fellows", 
+        "2012", 
```

Comment:
Presumably the tag `2012` is a year? What is the significance of 2012? When it was built? When it was deployed? If you want the fellowship year, that was 2011. I’m not sure such unstructured data is valuable.